R/plugin-awesomeMarkers.R

```diff
@@ -0,0 +1,187 @@
+leafletAwesomeMarkersDependencies <- function() {
+  list(
+    htmltools::htmlDependency(
+      "leaflet-awesomemarkers",
+      "2.0.2",
+      system.file("htmlwidgets/plugins/Leaflet.awesome-markers", package = "leaflet"),
+      script = c('leaflet.awesome-markers.min.js','bootstrap.min.js'),
+      stylesheet = c('leaflet.awesome-markers.css','bootstrap.min.css',
```

Comment:
The bootstrap and font-awesome dependencies need to be broken out in to separate `htmlDependency` objects. Otherwise this will break dependency resolution when leaflet is combined with Shiny.

This is what the htmlDependency objects look like in Shiny:
https://github.com/rstudio/shiny/blob/3d5c184acce71290533e8b3627638fca67fd9fd8/R/bootstrap.R#L39-L52
https://github.com/rstudio/shiny/blob/3d5c184acce71290533e8b3627638fca67fd9fd8/R/bootstrap.R#L1340-L1343

Let me know if you don't understand, I can make the change if you need me to.